hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/storage/HoodieConcatHandle.java

```diff
@@ -94,4 +107,34 @@ public void write(GenericRecord oldRecord) {
     }
     recordsWritten++;
   }
+
+  boolean needsUpdateLocation() {
```

Comment:
do we need this? already in merge handle this is set to true.

Reply:
It is because it has `package-private` access modifier; `HoodieMergeHandle` and `HoodieConcatHandle` are in different package. I can either move to the same package, or change the access modifier to `protected`.